test/test_fake_tensor.py

```diff
@@ -763,6 +763,8 @@ def run_meta():
         with patch.object(torch._functorch.config, "fake_tensor_allow_meta", False):
             self.assertRaises(Exception, run_meta)
 
+    # FakeTensor device vs. cuda implementations have mismatched size in 2nd output of _embedding_bag op
+    @expectedFailurePropagateRealTensors
```

Comment:
We're running this test with cpu FakeTensors, but for real tensors the device defaults to "cuda", and the offsets2bag output (which is actually not returned in the `torch.nn.EmbeddingBag.forward`, just the intermediate `aten._embedding_bag.default` op), has a different size b/w the cpu & cuda implementations:

https://github.com/pytorch/pytorch/blob/9f81270d7589fd7fa98dc247ae4b1b7ab239ca3c/torch/_meta_registrations.py#L3423-L3435

Reply:
yeah, you can't just default it. need to actually have the correct devices